bin/travis_setup.sh

```diff
@@ -13,26 +18,20 @@ if [ "$(uname)" == "Darwin" ]; then
 
 else
 
-    # Install LLVM/Clang 3.7, Boehm GC, libunwind
-    sudo apt-get -qq update
-    sudo sh -c "echo 'deb http://apt.llvm.org/precise/ llvm-toolchain-precise-3.7 main' >> /etc/apt/sources.list"
-    sudo sh -c "echo 'deb http://apt.llvm.org/precise/ llvm-toolchain-precise main' >> /etc/apt/sources.list"
-    wget -O - http://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+    export LLVM_VERSION=${LLVM_VERSION:-3.8}
```

Comment:
I can remove it, but I thought the idea was to make this script usable by external projects, such as https://github.com/scala-native/scala-native.g8.